plugins/available/osx.plugin.bash

```diff
@@ -111,5 +111,78 @@ function refresh-launchpad() {
   fi
 }
 
+function list-jvms(){
+  about 'List java virtual machines and their states in macOS'
+  example 'list-jvms'
+  group 'osx'
+
+  JDKS_DIR="/Library/Java/JavaVirtualMachines"
+  JDKS=( $(ls ${JDKS_DIR}) )
+  JDKS_STATES=()
+
+  # Map state of JDK
+  for (( i = 0; i < ${#JDKS[@]}; i++ )); do
+    if [[ -f "${JDKS_DIR}/${JDKS[$i]}/Contents/Info.plist" ]]; then
+      JDKS_STATES[${i}]=enable
+    else
+      JDKS_STATES[${i}]=disable
+    fi
+    echo "${i} ${JDKS[$i]} ${JDKS_STATES[$i]}"
+  done
+}
+
+
+function pick-default-jvm(){
+  about 'Pick a default java virtual machines in system-wide scope in macOS'
+  example 'pick-default-jvm'
+
+  JDKS_DIR="/Library/Java/JavaVirtualMachines"
```

Comment:
This whole section is an exact copy of the `list-jvms` function - please call this function instead of copy/paste.